src/store/state-url-mapping.ts

```diff
@@ -69,6 +73,41 @@ export const paramSetup = {
         selectedConsumer: {
             stateKey: 'partitions.selectedConsumer',
         },
+        selectedPartition: {
```

Comment:
Please, do not add new params to `redux-locations-state`, it makes harder to delete it https://github.com/ydb-platform/ydb-embedded-ui/issues/2135.

Use `use-query-params` hooks instead

Reply:
fixed!